docs/get-started/prototyping.md

```diff
@@ -5,28 +5,27 @@ tags:
   - getStarted
 ---
 
-This guide explains how to create prototypes using the NHS App design system and NHS App prototype, so you can make interactive prototypes that will look like pages on the NHS App.
+This guide explains how to use the NHS App prototype. You can use it alongside the NHS App design system to make interactive prototypes that look like real NHS App pages.
 
-## Before you start
-
-The NHS App prototype is a copy of the NHS prototype kit, with added NHS App code and templates.
+Anyone can use the prototype to:
 
-You must follow the [NHS prototype kit setup guide](https://prototype-kit.service-manual.nhs.uk/install) first.
+- try out ideas
+- get feedback from users or people you work with
 
-## Download
+## Before you start
 
-Download or clone a copy of the [NHS App prototype on GitHub](https://github.com/nhsuk/nhsapp-prototype).
+You first need to follow the [NHS prototype kit setup guide](https://prototype-kit.service-manual.nhs.uk/install) to make sure you have the tools and software required to run the prototype. This is because the NHS App prototype is a copy of the NHS prototype kit, with added NHS App code and templates.
 
 ## Guides
```

Comment:
The 'get started' page while not a full tutorial, is a form of tutorial itself. It is the first exposure the user would have to doing a self-help installation and use of the kit.

Part of the reason I made the suggestion is also because I have recently used Apple & Google's guidance as well and it follows the pattern I mentioned prior. It started with little steps and wins which made me feel like I was slowly building towards something.

Essentially, you are slowly exposing the user to concepts such as installing, trying it out (rewarding them with a successful trial) and then sending them to the wider resource. The guide being in the middle disrupts this momentum because the guide is broad and what the user need here is something more focused, little steps and little wins that helps their momentum and builds their confidence.

It makes sense for them to copy the code snippet on this page and add it to their kit, so they see the instant result of how it works, they get a joyful feeling and their confidence grows. Now, then they can go to the wider guides to see what else is there.

Reply:
Ok, makes sense, so:

- Learn what the prototype kit is
- Get the tools/software required so I can use it (by following the NHS prototype install guide)
- Start using the NHS App prototype (by downloading a copy)
- Try out a sample component
- Understand more about how to use it (by following the NHS prototype kit guides)
- Get support if needed